authors/apps/profiles/urls.py

```diff
@@ -1,14 +1,19 @@
 from django.urls import path
+
 from .views import (
     UserProfileView,
     UserProfileUpdateView,
     UserProfileListView,
+    FollowsUpdateAPIView,
+    MyFollowersListView,
+    IFollowListView,
 )
 
-
 urlpatterns = [
     path('<username>', UserProfileView.as_view(), name='profile-detail'),
     path('<username>/edit', UserProfileUpdateView.as_view(), name='profile-update'),
     path('', UserProfileListView.as_view(), name='profile-list'),
-
+    path('follow/', IFollowListView.as_view(), name='get_i_follow'),
```

Comment:
These endpoints are going to conflict with the profile endpoints.
The `follow` endpoint will be considered as a username.
Let's use the following ...
- follow a user  `<username>/follow`
- unfollow a user  `<username>/unfollow`
- list of followers  `<username>/followers`

Reply:
@manueldominic, use always use the readme as a reference.